contrib/babelfishpg_tsql/sql/babelfishpg_tsql.sql

```diff
@@ -459,6 +464,9 @@ CREATE OR REPLACE PROCEDURE sys.sp_columns_100 (
     "@fusepattern" smallint = 1)
 AS $$
 BEGIN
+	-- TODO: we should be able to get rid of babelfish_truncate_identifier when we fix BABEL-5416
+	declare @truncated_ident sys.nvarchar(384);
+	select @truncated_ident = sys.babelfish_truncate_identifier(pg_catalog.lower(@table_name));
```

Comment:
sys.babelfish_truncate_identifier only truncates the input. So it is expected that we pass lower case input so that sys.babelfish_truncate_identifier does expected work.